inetum-realjobs-frontend/src/app/app.routing.ts

```diff
@@ -14,6 +15,7 @@ const appRoutes: Routes = [
   {path: "register", component: RegisterComponent},
   {path: "vacancy/search", component: FilterFormComponent},
   {path: "vacancy/:id", component: VacancyDetailsComponent},
+  {path: "resume/create", component: ResumeCreateComponent, canActivate: [AuthGuardService]},
```

Comment:
Shouldn't it be only available to job seekers? This guard only requires authentication iirc